src/v2/components/BlockLightbox/components/BlockLightboxLink/components/BlockLightboxLinkScreenshot/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Box from 'v2/components/UI/Box'
+import Text from 'v2/components/UI/Text'
+import Icons from 'v2/components/UI/Icons'
+
+import { BlockLightboxLinkProps } from 'v2/components/BlockLightbox/components/BlockLightboxLink'
+
+const Screenshot = styled(Box)``
+
+const Image = styled(Box)`
+  position: relative;
+  background-size: cover;
+  background-position: top center;
+  background-image: url(${props => props.backgroundImage});
```

Comment:
I'd set this with `style` I'm sure we've messed this up in other places but generally consider that styled-components generates classes, so it would generate a new class for every single possible input on a page here. In this case there's only one on screen at a time so... nbd. But *in general* only pass props where you've got a finite set of possibilities.

Reply:
Ah got it. Will fix that. This is mostly just a renaming of this file, to .tsx, but it's a good catch.